codecov.yml

```diff
@@ -1,5 +1,6 @@
 fixes:
-  - "src/navigation2/::"
+  - "**/src/navigation2/::"
```

Comment:
I made this use a glob pattern to fix absolute file paths. Not sure why a directory globing wasn't used before.